cognite/client/_api/data_modeling/instances.py

```diff
@@ -532,7 +532,7 @@ def search(
         instance_type: Literal["node"] = "node",
         properties: list[str] | None = None,
         filter: Filter | dict | None = None,
-        limit: int = INSTANCES_LIST_LIMIT_DEFAULT,
+        limit: int = 1000,
```

Comment:
I agree with Erlend here. I think I did a mistake here by introducing `INSTANCES_LIST_LIMIT_DEFAULT = 1000`, I think instead this one should be set to `LIST_LIMIT_DEFAULT`  thus 25. And then let the user have to explicitly set limit if they want more.

Reply:
> Why are we changing these to not be constants?

Because that is the de facto standard in the SDK:
```
# Defaults:
[('AssetsAPI    , list limit: 25, search limit: 100',),
 ('DocumentsAPI , list limit: 25, search limit: 25',),
 ('EventsAPI    , list limit: 25, search limit: 100',),
 ('FilesAPI     , list limit: 25, search limit: 100',),
 ('SequencesAPI , list limit: 25, search limit: 100',),
 ('TimeSeriesAPI, list limit: 25, search limit: 100',)]
```
All 25's are `LIST_LIMIT_DEFAULT` and all 100s are literals.

> The default should not be the max btw - it encourages abuse of the API. Most use cases only need the first few results

Very much agree, but my change left the value unchanged, `INSTANCES_LIST_LIMIT_DEFAULT` is set to 1000.

Reply:
I guess my real question here is: why shouldn't just all endpoints default to something small like 25? 🤔 What is the logic behind 100 for search and 25 for list? 🤔

Reply:
Personally, when using search, I typically need/use 1-5 top entries